readthedocs/api/v2/views/integrations.py

```diff
@@ -409,12 +409,7 @@ def handle_webhook(self):
             return self.sync_versions_response(self.project)
 
         # Handle pull request events
-        if all([
-                self.project.has_feature(Feature.EXTERNAL_VERSION_BUILD),
-                self.project.external_builds_enabled,
-                event == GITHUB_PULL_REQUEST,
-                action,
```

Comment:
action is already checked with `action in [one, two]` so there is no need to check it here